using_the_compiler/README.md

```diff
@@ -360,3 +360,12 @@ crystal tool format [--check] [<path>...]
 
 `path` can be a file or folder name and include all Crystal files in that folder tree. Omitting `path` is equal to
 specifying the current working directory.
+
+## Environment variables
+
+The following environment variables are used by the Crystal compiler if set in the environment. Otherwise the compiler will populate them with default values.. Their values can be inspected using `[crystal env](#crystal env)`.
```

Comment:
Remove a redundant dot and fix `crystal env` link:
```suggestion
The following environment variables are used by the Crystal compiler if set in the environment. Otherwise the compiler will populate them with default values. Their values can be inspected using `[crystal env](#crystal-env)`.
```